support-frontend/assets/pages/paper-subscription-landing/components/hero/discountCopy.jsx

```diff
@@ -14,11 +14,11 @@ const discountCopy = (discountPercentage: number): DiscountCopy => (
   discountPercentage > 0 ?
     {
       roundel: ['Save up to', `${discountPercentage}%`, 'a year'],
-      heading: `Save up to ${discountPercentage}% a year with a subscription`,
+      heading: [`Save up to ${discountPercentage}% a year with a subscription`],
```

Comment:
from the type definition above - should this just be a string?
`heading: string | Node,`

Reply:
Yes, you're right it should. Thanks